lib/hanami/configuration.rb

```diff
@@ -77,6 +114,14 @@ def logger
       settings.fetch(:logger)
     end
 
+    def rack_logger_filter_params=(params)
+      settings[:rack_logger_filter_params] = params
+    end
+
+    def rack_logger_filter_params
+      settings[:rack_logger_filter_params]
+    end
+
```

Comment:
I'm aware these (and the filtering behaviour in `RackLogger`) duplicate existing functionality in `Hanami::Logger` with its (seemingly not documented?) `filters:` option. However, for the sake of expedience, I'd like it if we could just accept these for now and then look at reconciling these different loggers post-merge.

Reply:
Yep, this is noted 👍